sql/core/src/test/scala/org/apache/spark/sql/execution/arrow/ArrowConvertersSuite.scala

```diff
@@ -1208,19 +1208,6 @@ class ArrowConvertersSuite extends SharedSparkSession {
     spark.conf.unset(SQLConf.ARROW_EXECUTION_MAX_RECORDS_PER_BATCH.key)
   }
 
-  testQuietly("unsupported types") {
```

Comment:
Can we keep this test by using another type? (e.g., CalendarIntervalType or UDT).

Reply:
Done, modified this test to work with CalendarInternvalType